app/Http/Requests/Api/RolesRequest.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace App\Http\Requests\Api;
+
+use Illuminate\Foundation\Http\FormRequest;
+
+class RolesRequest extends FormRequest
```

Comment:
the file and class name should be `RoleRequest` because this will always process one role